crates/core/src/db/queries/stats.rs

```diff
@@ -139,3 +139,30 @@ pub fn collection(
         .load(conn)
         .context("Failed to load collection mint stats")
 }
+
+const STORE_CREATOR_QUERY: &str = r"
+select
+    sc.creator_address as store_creator,
+    count(distinct mc.metadata_address)::bigint as nfts
+
+from store_creators sc
+    inner join metadata_creators mc
+        on (mc.creator_address = sc.creator_address)
+
+where sc.creator_address = any($1)
```

Comment:
```suggestion
where sc.creator_address = any($1) AND mc.verfied
```

Only count if the metadata_creator is verified on the nft.